src/Utils.php

```diff
@@ -0,0 +1,35 @@
+<?php namespace Lean\Elements;
+
+/**
+ * Class Utils
+ *
+ * @package Lean\Elements
+ */
+class Utils
+{
+	/**
+	 * Convert a "watch" url into an "embed"
+	 * Eg https://www.youtube.com/watch?v=OQZKh8Bjdv0 to https://www.youtube.com/embed/OQZKh8Bjdv0
+	 * Or https://vimeo.com/155086124 to https://player.vimeo.com/video/155086124
+	 *
+	 * @param string $url The video url.
+	 * @return string
+	 */
+	public static function get_video_embed_url( $url ) {
```

Comment:
They're similar, but subtly different. I'd prefer to keep as one function, rather than have to determine if it's yourtube or vimeo before deciding which function to call.